lib/ClassMover/Adapter/WorseTolerant/WorseTolerantMemberFinder.php

```diff
@@ -270,7 +271,7 @@ private function getMemberDeclarationReference(ReflectionClassLike $queryClass =
 
         // if the references class is not an instance of the requested class, or the requested class is not
         // an instance of the referenced class then ignore it.
-        if (false === $reflectionClass->isTrait() && false === $reflectionClass->isInstanceOf($queryClass->name())) {
```

Comment:
Psalm has `@psalm-assert-if-true ReflectionTrait $this` or similar, to make "query" conditions like this work. Not in PHPStan currently though AFAIK.

Reply:
Yeah, but since the `isTrait()` function already was deprecated I just replaced it or wasn't it meant to be removed yet?

Reply:
No all good :+1: